nilearn/plotting/js_plotting_utils.py

```diff
@@ -147,7 +147,7 @@ def save_as_html(self, file_name):
         Save the plot in an HTML file, that can later be opened in a browser.
         """
         with open(file_name, 'wb') as f:
-            f.write(self.html.encode('utf-8'))
+            f.write(self.get_standalone().encode('utf-8'))
```

Comment:
this change is fine but please also add an `html` property to the reports for compatibility (comment below)